homeassistant/components/proxmoxve/__init__.py

```diff
@@ -16,20 +16,23 @@
 )
 import homeassistant.helpers.config_validation as cv
 
-_LOGGER = logging.getLogger(__name__)
+from .const import (
+    CONF_CONTAINERS,
+    CONF_NODE,
+    CONF_NODES,
+    CONF_REALM,
+    CONF_VMS,
+    DEFAULT_PORT,
+    DEFAULT_REALM,
+    DEFAULT_VERIFY_SSL,
+    DOMAIN,
+    PROXMOX_CLIENTS,
+)
 
+_LOGGER = logging.getLogger(__name__)
 
-DOMAIN = "proxmoxve"
-PROXMOX_CLIENTS = "proxmox_clients"
-CONF_REALM = "realm"
-CONF_NODE = "node"
-CONF_NODES = "nodes"
-CONF_VMS = "vms"
-CONF_CONTAINERS = "containers"
 
-DEFAULT_PORT = 8006
-DEFAULT_REALM = "pam"
-DEFAULT_VERIFY_SSL = True
+PLATFORMS = ["binary_sensor"]
 
 CONFIG_SCHEMA = vol.Schema(
```

Comment:
Please deprecate the config schema and import the current configuration via the config flow.